packages/babel-generator/src/generators/methods.js

```diff
@@ -77,8 +77,14 @@ export function ArrowFunctionExpression(node: Object) {
     this.space();
   }
 
-  if (node.params.length === 1 && t.isIdentifier(node.params[0])) {
-    this.print(node.params[0], node);
+  let param;
+
+  if (
+    node.params.length === 1
+    && t.isIdentifier(param = node.params[0])
```

Comment:
ah yeah, idk why I didn't assign it immediately. I've pushed a change for that (and extracted the type checking into a function to be explicit).